apps/user-group-cdc/src/main/java/it/pagopa/selfcare/user/event/UserGroupCdcService.java

```diff
@@ -0,0 +1,162 @@
+                               @ConfigProperty(name = "user-group-cdc.retry") Integer maxRetry,
+                               @ConfigProperty(name = "user-group-cdc.send-events.watch.enabled") Boolean sendEventsEnabled,
+                               TelemetryClient telemetryClient,
+                               TableClient tableClient,
+
+                               UserGroupNotificationMapper userGroupNotificationMapper) {
+        this.maxRetry = maxRetry;
+        this.mongoClient = mongoClient;
+        this.mongodbDatabase = mongodbDatabase;
+        this.retryMaxBackOff = retryMaxBackOff;
+        this.retryMinBackOff = retryMinBackOff;
+        this.telemetryClient = telemetryClient;
+        this.userGroupNotificationMapper = userGroupNotificationMapper;
+        this.tableClient = tableClient;
+        telemetryClient.getContext().getOperation().setName(OPERATION_NAME);
+        initOrderStream(sendEventsEnabled);
+    }
+
+    private void initOrderStream(Boolean sendEventsEnabled) {
+        log.info("Starting initOrderStream ... ");
+
+        String resumeToken = null;
+
+        if (!ConfigUtils.getProfiles().contains("test")) {
+            try {
+                TableEntity cdcStartAtEntity = tableClient.getEntity(CDC_START_AT_PARTITION_KEY, CDC_START_AT_ROW_KEY);
+                if (Objects.nonNull(cdcStartAtEntity))
+                    resumeToken = (String) cdcStartAtEntity.getProperty(CDC_START_AT_PROPERTY);
+            } catch (TableServiceException e) {
+                log.warn("Table StarAt not found, it is starting from now ...");
+            }
+        }
+
+        ReactiveMongoCollection<UserGroupEntity> dataCollection = getCollection();
+        ChangeStreamOptions options = new ChangeStreamOptions()
+                .fullDocument(FullDocument.UPDATE_LOOKUP);
+        if (Objects.nonNull(resumeToken))
+            options = options.resumeAfter(BsonDocument.parse(resumeToken));
+
+        Bson match = Aggregates.match(Filters.in("operationType", asList("update", "replace", "insert")));
+        Bson project = Aggregates.project(fields(include("_id", "ns", "documentKey", "fullDocument")));
+        List<Bson> pipeline = Arrays.asList(match, project);
+
+        Multi<ChangeStreamDocument<UserGroupEntity>> publisher = dataCollection.watch(pipeline, UserGroupEntity.class, options);
+
+        if (sendEventsEnabled) {
+            publisher.subscribe().with(
+                    this::consumerToSendScUserGroupEvent,
+                    failure -> {
+                        log.error("Error during subscribe collection, exception: {} , message: {}", failure.toString(), failure.getMessage());
+                        telemetryClient.trackEvent(EVENT_USER_GROUP_CDC_NAME, mapPropsForTrackEvent(TrackEventInput.builder().exception(failure.getClass().toString()).build()), Map.of(EVENTS_USER_INSTITUTION_FAILURE, 1D));
```

Comment:
EVENTS_USER_INSTITUTION_FAILURE ? you should use a new event about USER_GROUP_FAILURE